index.html

```diff
@@ -280,28 +222,63 @@ <h4 class="frame-name">CSS</h4>
             <li class="tabs">
               <h4 class="tab-title">Framework</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
           <ul class="pro-languages separator-bottom">
             <li class="tabs">
               <h4 class="tab-title">Skills</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
         </div>
       </div>
     </section>
+    <section class="form">
+      <ul class="form-header">
+        <li class="form-list">
+          <h4 class="contact-title">Contact me</h4>
+          <p class="contact-description">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it.
+          </p>
+        </li>
+      </ul>
+      <form action="https://formspree.io/f/mgebvkpz" method="POST"  class="form-details">
+        <input
+        name="name"
+          class="username"
+          type="text"
+          id="name"
+          placeholder="Yeremias"
+          maxlength="30"
+          required
+        />
+        <input 
+        name="email"
+         class="useremail"
+          type="email"
+          id="email"
+          placeholder="natajayanj14@gmail.com"
+          required
+        />
+        <textarea
+        class="userfeedback"
+          name="feedback"
+          id="feedback"
+          cols="30"
+          rows="10"
+          maxlength="500"
+          placeholder="Write your message here"
+          required
+        ></textarea>
```

Comment:
As the previous reviewer requested add padding to the left side of the input texts. Kindly check the padding value on the template.

<img width="355" alt="Screenshot 2023-01-17 at 9 54 26 PM" src="https://user-images.githubusercontent.com/25887253/212988394-15e620a9-9e7c-40b7-a9f2-11dcbc7fc78b.png">
<img width="355" alt="Screenshot 2023-01-17 at 9 54 44 PM" src="https://user-images.githubusercontent.com/25887253/212988402-e82d1e68-dc4c-4bad-ba70-eeacfadd0fb1.png">